### PR TITLE
Bugfix : [Dark Mode] Custom macro field gets unreadable when edited

### PR DESCRIPTION
**Pull request by ataghzout**
## Description
Fixbug custom macro field and apply dark mode  correctly 
image

**Fixes** # MON-13195

## Type of change

- [X] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [X] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [X] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [X] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by kduret — DISMISSED** [review]

**Review by callapa — DISMISSED** [review]

**Review by jeremyjaouen — DISMISSED** [review]

File: www/include/configuration/configObject/service/formService.ihtml [hunk 1]
```diff
@@ -621,22 +633,34 @@ jQuery(function() {
                             jQuery(elem).find("input[name^='macroValue']").val(tplValue);
                             jQuery(elem).find("input[name^='macroInput']").val(tplInput);
                             jQuery(elem).find('input[name^=macroFrom]').val("fromTpl");
-                            jQuery(elem).find("input[name^='macroInput']").css({'background-color' : '#ffcb6b',border : '1px solid #ffa600'});
-                            jQuery(elem).find("input[name^='macroValue']").css({'background-color' : '#ffcb6b',border : '1px solid #ffa600'});
+                            jQuery(elem).find("input[name^='macroInput']").css({'background-color' : '#ffcb6b',border : '1px solid #ffa600',color: 'var(--custom-macros-placeholder-font-color)'});
```
Comment (callapa) [inline www/include/configuration/configObject/service/formService.ihtml:636]: It is possible to use a variable for the color #ffa600 ?

Comment (ataghzout) [reply www/include/configuration/configObject/service/formService.ihtml:636]: @callapa it was like this before but yes it is possible to export it ! :)

Comment (callapa) [reply www/include/configuration/configObject/service/formService.ihtml:636]: That was the purpose of the "dark mode" feature, right?

File: www/include/configuration/configObject/host/formHost.ihtml [hunk 2]
```diff
@@ -598,22 +598,34 @@ jQuery(function() {
...
-            jQuery(elem).find("input[name^='macroInput']").css("background","white");
-            jQuery(elem).find("input[name^='macroValue']").css("background","white");
+			jQuery(elem).find("input[name^='macroInput']").css({
+				"color" : "var(--custom-macros-font-color)",
+				"background" : "var(--input-select-background-color)"
+			});
+			jQuery(elem).find("input[name^='macroValue']").css({
+				"color" : "var(--custom-macros-font-color)",
+				"background" : "var(--input-select-background-color)"
+			});
```
Comment (kduret) [inline www/include/configuration/configObject/host/formHost.ihtml:628]: do not use tab for indentation

File: www/include/configuration/configObject/host/formHost.ihtml [hunk 3]
```diff
@@ -598,22 +598,34 @@ jQuery(function() {
...
```
Comment (kduret) [inline www/include/configuration/configObject/host/formHost.ihtml:617]:
```suggestion
        jQuery(elem).find("input[name^='macroInput']").css({
            "color" : "var(--custom-macros-font-color)",
            "background" : "var(--input-select-background-color)"
        });
        jQuery(elem).find("input[name^='macroValue']").css({
            "color" : "var(--custom-macros-font-color)",
            "background" : "var(--input-select-background-color)"
        });
```

File: www/include/configuration/configObject/host/formHost.ihtml (hunk 2 above)
Comment (kduret) [inline www/include/configuration/configObject/host/formHost.ihtml:628]:
```suggestion
        jQuery(elem).find("input[name^='macroInput']").css({
            "color" : "var(--custom-macros-font-color)",
            "background" : "var(--input-select-background-color)"
        });
        jQuery(elem).find("input[name^='macroValue']").css({
            "color" : "var(--custom-macros-font-color)",
            "background" : "var(--input-select-background-color)"
        });
```

**Review by kduret — DISMISSED** [review]

**Review by kduret — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Pull request merged**